tools/cldr-code/src/main/java/org/unicode/cldr/test/SubmissionLocales.java

```diff
@@ -178,19 +172,23 @@ public static Enum<Level> getCoverageLevel(String localeString, String path) {
 
     /**
      * Only call this if LIMITED_SUBMISSION
+     *
      * @param localeString
      * @param path
      * @param isError
      * @param isMissing
+     * @param voteStatus
      * @return true if submission is allowed, else false
      */
-    public static boolean allowEvenIfLimited(String localeString, String path, boolean isError, boolean isMissing) {
+    public static boolean allowEvenIfLimited(String localeString, String path, boolean isError, boolean isMissing, VettingViewer.VoteStatus voteStatus) {
 
         // Allow errors to be fixed
         if (isError) {
             return true;
         }
-
+        if (voteStatus == VettingViewer.VoteStatus.provisionalOrWorse) {
+            return true;
```

Comment:
Could this change result in a very large increase in the number of paths allowed during limited submission, and if so, would that be appropriate?

Reply:
I think, and maybe we should put this up for discussion in tomorrow's meeting, but I think this check should go after the check for locales, because otherwise, it will allow provisional input in all locales, and I think that's too many for what we generally have been trying to do here

Reply:
That is right. The Provisional items should act like Missing, and thus only allow entry in specified locales (TC + exceptions)

Reply:
'specified locales' - should move to after line 207/206 where there's the `isMissing` check. Prior lines verify which locale it is.

Reply:
> https://github.com/unicode-org/cldr/pull/2698/files#diff-d5d790ecce246b7bc3d9b290e5f4e13fde07e2c0ca6b500363d8c984ccb6ed7fL207